### ISSUE TITLE
HDF5-DIAG: Error detected in HDF5 

### ISSUE THREAD

**Issue by Ecogpr** [jts/nanopolish#52]
I have a bit of an issue but it appears it can find the FAST5 files not sure what’s wrong.

$`python scripts/nanopolish_makerange.py miri.contigs.fasta | parallel --results nanopolish.results -P 8 nanopolish consensus -o nanopolish.{1}.fa -w {1} --r miriraw.pp.fa -b miriraw.pp.sorted.bam -g miri.contigs.fasta -t 12`
HDF5-DIAG: Error detected in HDF5 (1.8.16) thread 139649602275264:
  #000: ../../../src/H5A.c line 1692 in H5Aexists_by_name(): unable to determine if attribute exists
    major: Attribute
    minor: Can't get value
  #001: ../../../src/H5Aint.c line 1200 in H5A_exists_by_name(): object not found
    major: Attribute
    minor: Object not found
  #002: ../../../src/H5Gloc.c line 430 in H5G_loc_find(): can't find object
    major: Symbol table
    minor: Object not found
  #003: ../../../src/H5Gtraverse.c line 861 in H5G_traverse(): internal path traversal failed
    major: Symbol table
    minor: Object not found
  #004: ../../../src/H5Gtraverse.c line 755 in H5G_traverse_real(): component not found
    major: Symbol table
    minor: Object not found
terminate called after throwing an instance of 'hdf5_tools::Exception'
  what():  /Analyses/Basecall_2D_002/BaseCalled_template/Model: error in H5Aexists_by_name

**Comment by jts** [comment]
Hi,

This problem might be caused by using an out-of-date version of nanopolish. Can you check whether the problem persists after updating to the HEAD on github (and recompilng with `make clean && make`)? If you still have an error we'll probably need to see a few FAST5 files to debug.

Thanks,
Jared

**Comment by Ecogpr** [comment]
Jared,

Thanks, `make clean && make` worked for me.  Straight from `git clone` to `make` didn’t create the executable in my directory, don’t know why.  I had an older deb package that had its own terminal command set for it.

Once I removed that package and `make clean && make` in the `git clone` folder, I got the executable.  Then I changed my command to

`python scripts/nanopolish_makerange.py miri.contigs.fasta | parallel --results nanopolish.results -P 8 ./nanopolish consensus -o nanopolish.{1}.fa -w {1} --r miriraw.pp.fa -b miriraw.pp.sorted.bam -g miri.contigs.fasta -t 12`

Now it is working.

Thanks
